dist/index.js

```diff
@@ -26,107 +26,119 @@ var __importStar = (this && this.__importStar) || function (mod) {
     __setModuleDefault(result, mod);
     return result;
 };
-var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, generator) {
```

Comment:
should dist be added to or excluded from the repo?

Reply:
nvm, dist appears to be its own thing.

Reply:
@annlin-xealth eventually we will make our own CI process for releasing this action, including packaging the `dist` directory and releasing it via artifacts. That is the next step, though.

Reply:
@annlin-xealth Resolving this convo as we've implemented the CI process.